tests/util/pilot_server.go

```diff
@@ -130,6 +130,13 @@ func setup() error {
 	}
 	pilotHTTPPort, _ := strconv.Atoi(pilotHTTP)
 
+	_, meshConfig, _ := bootstrap.GetMeshConfig(nil, "not used", "not used")
+	meshConfig.AccessLogFile = "/tmp/envoy-access.log" // TODO take value from TestSetup.AccessLogPath, needs refactoring to avoid import cycle
+	//TODO: start mixer first, get its address
+	meshConfig.MixerCheckServer = "istio-mixer.istio-system:9091"
```

Comment:
Didn't `istio-mixer.istio-system` got split into `istio-telemetry.istio-system` (report) and `istio-policy.istio-system`  (check) ?

Reply:
Oh, cool! This means that this refactoring would allow now to adapt the test set up to that. Not sure though if we should mix the change of the URLs with this one immediately or if that isn't a separate change.